opentelemetry-appender-log/Cargo.toml

```diff
@@ -18,6 +18,7 @@ serde = { workspace = true, optional = true, features = ["std"] }
 [features]
 logs_level_enabled = ["opentelemetry/logs_level_enabled"]
 with-serde = ["log/kv_serde", "serde"]
+default = ["logs_level_enabled", "with-serde"]
```

Comment:
is this needed?

Reply:
Oh, this was only for testing. Good catch. Let me remove it.